### PR TITLE
Node port

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

**Comment by yuval-k** [comment]
blocked by https://github.com/solo-io/gloo/pull/396

File: docs/getting_started/kubernetes/node_port.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+By default, micro-services deployed in kubernetes have an internal flat network that is not accessible from the outside
+of the cluster. This is true even if you use kuberentes on a public cloud (like Amazon AWs or Google Cloud).
+
+A NodePort is a way to make kuberentes services available from outside the cluster (and potentially allow access from the internet).
+
+In this document we will review how to expose Gloo via a NodePort.
+
+
+## What is NodePort Service?
+A kuberenetes cluster is composed of one or more nodes. a node vm is a linux (most likely) machine (can be a virtual machine or bare-metal) that actually runs the kuberenetes pods.
```
Comment (rickducott) [inline docs/getting_started/kubernetes/node_port.md]: A node VM...

File: docs/getting_started/kubernetes/node_port.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+A NodePort is a way to make kuberentes services available from outside the cluster (and potentially allow access from the internet).
+
+In this document we will review how to expose Gloo via a NodePort.
+
+
+## What is NodePort Service?
+A kuberenetes cluster is composed of one or more nodes. a node vm is a linux (most likely) machine (can be a virtual machine or bare-metal) that actually runs the kuberenetes pods. 
+
+When a kubernetes service is created with NodePort type, kubernetes chooses port number and assigns it to the service. In addition, every node in the cluster is configured to forward traffic from this port to the pods belonging service.
```
Comment (rickducott) [inline docs/getting_started/kubernetes/node_port.md]: kubernetes chooses *a* port number

File: docs/getting_started/kubernetes/node_port.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+This allows you to access the service simply by connecting to a 'node-ip:node-port' where node-ip is the ip of any node in the cluster, and node-port is the NodePort assigned by kuberentes.
+
+One advantage of using a NodePort is that it allows relatively easy deployment on bare metal, as it does not depend on any load balancing component outside the cluster.
+
+## How to use Gloo with NodePort?
+
+In Gloo, the service that's responsible ingress data is called "gateway-proxy". To use Gloo with NodePort
```
Comment (rickducott) [inline docs/getting_started/kubernetes/node_port.md]: ...that's responsible **for** ingress...

Comment (yuval-k) [reply docs/getting_started/kubernetes/node_port.md]: also changed data to traffic

**Pull request merged**